lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -13,45 +13,66 @@
 }
 
   @mixin dropdown-link() {
+    @include box-sizing(border-box);
+    @include font-size(-2)
     position: relative;
+    padding: $baseline/2.5 $baseline/1;
     display: block;
-    @include box-sizing(border-box);
-    padding: $baseline/3;
-    @include font-size(-2);
     color: $darker-gray;
-    background-color: $white;
-    border-bottom: none;
+    font-weight: inherit;
```

Comment:
I think you could just remove this, yeah? Or was there a reason you have to explicitly declare it?

Reply:
Had removed them on the latest commit, sorry.